modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -181,11 +181,13 @@
                           (* 24 60 60 1000)]}]})
 
 (defn- truncate-to-resolution [column resolution]
```

Comment:
if we're making everything explicit wrt timezones I wonder if it's worth the while to (programmatically) note in the line 121 `->rvalue` for fields that all the unix timestamps will always be utc, no exceptions, despite any overlying let bits?

Reply:
I'm uncertain what to do (and therefore inclined not to change).  When converting UNIX epoch stuff, UTC is seems the right choice, so we could specify it (as a kind of documentation), but the conversion of the string literal might in fact make more sense and cause fewer surprises in the result time zone. But setting the result time zone might hurt existing customers, so doing that without a bug report seems questionable. @howonlee WDYT? Under what circumstances do these conversions get triggered?

Reply:
Assuming UNIX epoch timestamp integers are all in UTC seems sane and it's what we're doing for all the other drivers. Not sure if there are any open bugs related to us _not_ doing that but either way we should probably still be doing it

Reply:
Added `:timezone "UTC"` for conversions from UNIX epoch timestamps.